PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle

```diff
@@ -94,13 +94,15 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'info.cukes:cucumber-junit:1.2.5'
     testCompile 'info.cukes:cucumber-java:1.2.5'
-    testCompile 'org.mockito:mockito-core:2.7.22'
+    testCompile 'org.mockito:mockito-core:2.8.47'
     testCompile 'org.assertj:assertj-swing-junit:3.5.0'
 
-    testCompile 'com.github.tomakehurst:wiremock:2.8.0'
-    testCompile 'org.powermock:powermock-module-junit4:1.7.0RC4'
-    testCompile 'org.powermock:powermock-api-mockito2:1.7.0RC4'
+    testCompile 'com.github.tomakehurst:wiremock:2.6.0'
```

Comment:
version downgrade reason?

Reply:
The cases can pass on travis-ci and my local machine. Not sure why it failed on jenkins. The suggestion on SO is this is a dependency version issue. So I am trying to find a suitable one.